lib/ledgerEventStorage.js

```diff
@@ -199,6 +197,9 @@ module.exports = class LedgerEventStorage {
     this.collection.find(query).count(callback);
   }
 
+  // FIXME: this seems like a very expensive operation and I don't think
+  // this API is in use anywhere, should it be removed?
```

Comment:
Good FIXME ... I suspect we could get rid of it. Or at least make it private and not expose any docs on it until we know we need it.